pinot-common/src/main/java/org/apache/pinot/common/request/context/RequestContextUtils.java

```diff
@@ -230,6 +231,9 @@ private static String getStringValue(Expression thriftExpression) {
       throw new BadQueryRequestException(
           "Pinot does not support column or function on the right-hand side of the predicate");
     }
+    if (thriftExpression.getLiteral().getSetField() == Literal._Fields.NULL_VALUE) {
+      return String.valueOf(null);
```

Comment:
We can simply return "null" here, right avoiding the branching inside `.valueOf` ?